index.js

```diff
@@ -122,6 +123,12 @@ function run(input, flags, showHelp) {
 
 	runWebpack(webpackConfig).then(
 		function(webpackRes) {
+			if (webpackRes.errors.length > 0) {
+				for (var i = 0; i < webpackRes.errors.length; i++) {
+					console.error(webpackRes.errors[i]);
+				}
+				return;
```

Comment:
You should also terminate the process with an error code to ensure that the test run was unsucessful.